src/assets/js/application.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-env browser */
+
+/* global ClipboardJS:false */
+
+import ClipboardJS from 'clipboard';
+
+const btnHtml = [
+'<div class="bd-clipboard">',
+  '<button type="button" class="btn-clipboard" title="Copy to clipboard">',
+    '<svg class="bi" width="1em" height="1em" fill="currentColor">',
+      '<use xlink:href="#clipboard"/>',
+    '</svg>',
+  '</button>',
+'</div>'].join('');
+
+document.querySelectorAll('div.highlight')
+  .forEach((element) => {
+    element.insertAdjacentHTML('beforebegin', btnHtml)
+  })
+
+const clipboard = new ClipboardJS('.btn-clipboard', {
+  target: trigger => trigger.parentNode.nextElementSibling,
+  text: trigger => trigger.parentNode.nextElementSibling.textContent.trimEnd()
+})
+
+clipboard.on('success', (event) => {
+  const iconFirstChild = event.trigger.querySelector('.bi').firstElementChild
```

Comment:
Switched to `firstElementChild` so that we are guaranteed to get the first element and not text

Reply:
Just out of curiosity, did you spot in this repo or in the main one an use case where it didn't work with `firstChild`?

Reply:
Yeah sure no problem with this modification of course, I've already tried it manually. I was just wondering if it was already a problem somewhere or just an enhancement :)